x-pack/legacy/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -695,22 +796,26 @@ describe('Lens App', () => {
       );
     });
 
-    it('updates the filters when the user changes them', () => {
+    it('updates the filters when the user changes them', async () => {
       const args = defaultArgs;
       args.editorFrame = frame;
 
       const instance = mount(<App {...args} />);
       const indexPattern = ({ id: 'index1' } as unknown) as IIndexPattern;
       const field = ({ name: 'myfield' } as unknown) as IFieldType;
 
-      args.data.query.filterManager.setFilters([esFilters.buildExistsFilter(field, indexPattern)]);
+      const filter = {
```

Comment:
Nitpick: Not sure if it's valuable (treat this comment as learning opportunity for me :D ) but in the query tests below we construct filters using setting Filter Store:
```
      const unpinned = esFilters.buildExistsFilter(field, indexPattern);
      FilterManager.setFiltersStore([unpinned], esFilters.FilterStateStore.APP_STATE);
```
while here you construct the filter object. Maybe it's worth to unify it?

Reply:
I looked into this a bit more and specifically the `exists` filter has a different type definition than the other types. This is definitely worth highlighting to so I'm building the filters manually!